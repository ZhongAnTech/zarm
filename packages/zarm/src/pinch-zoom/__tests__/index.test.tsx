/* eslint-disable @typescript-eslint/no-var-requires */
/* eslint-disable dot-notation */
import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import PinchZoom from '../index';

describe('PinchZoom', () => {
  describe('snapshot', () => {
    it('renders correctly', () => {
      const wrapper = render(<PinchZoom />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
