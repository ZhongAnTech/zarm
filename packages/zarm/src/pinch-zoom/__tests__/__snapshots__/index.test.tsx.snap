// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinchZoom snapshot renders correctly 1`] = `
<div
  class="za-pinch-zoom"
/>
`;
